src/app/Http/Controllers/PagesController.php

```diff
@@ -143,7 +143,7 @@ public function getQuizById($id)
 
     public function getUsers()
     {
-        if (request()->user()->can('view', User::class)){
```

Comment:
Is there a reason we're changing from this clean dynamic format to the more messy string format?

Reply:
It can be clean like this, but needs more such as allow Export, Review, Import which are not included from View, Create, Edit, Delete (and i saw those only added for admin). So I added just like what I proposed in the role-permission list that weeks ago. I could re-organise them if this is necessary.